src/index.ts

```diff
@@ -40,6 +40,52 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
+				return value.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data'
+			}
+
+			return containsMultipartOperation(value)
+		})
+
+		hbs.registerHelper('isMultipartSchema', function(value: CodegenSchema): boolean {
+			return value.originalName === 'multipart/form-data'
+		})
+
+		hbs.registerHelper('isMetadataSchema', function(value: CodegenSchema): boolean {
+			return value.purpose === CodegenSchemaPurpose.METADATA
+		})
+
+		hbs.registerHelper('getFileProperties', function(properties: CodegenObjectLikeSchemas[]): FileProperty[] {
```

Comment:
perhaps this should just be `fileProperties`... does that match other helper naming?

Reply:
I might need to rethink this one entirely. It needs to indicate it gets all the properties that are files.

Reply:
I feel like keeping the `get` in the helper name makes it obvious that it retrieves something.